app/assets/stylesheets/blacklight/_catalog.scss

```diff
@@ -100,6 +100,10 @@ span.constraints-label {
     padding-top: $padding-base-vertical;
     border-bottom:1px dotted $table-border-color;
     @extend .clearfix;
+  
+  .document-title-heading {
```

Comment:
Line should be indented 4 spaces, but was indented 2 spaces<br>Selector should have depth of applicability no greater than 2, but was 3